<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1497</width>
    <height>1140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aplikacja do analizy częstotliwościowej sygnału</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QTabWidget" name="tabWidget">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tabWidgetPage1">
         <attribute name="title">
          <string>Sygnał 1D</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="layout">
            <property name="verticalSpacing">
             <number>7</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <family>Century Gothic</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
font: 75 italic bold 12pt &quot;Century Gothic&quot;;}</string>
              </property>
              <property name="text">
               <string>DANE SYGNAŁU</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignHCenter|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QFrame" name="frame_2">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="styleSheet">
                  <string notr="true">QLabel{
font: 75 italic 10pt &quot;Century Gothic&quot;;}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="text">
                  <string>Amplituda:</string>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::AutoText</enum>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QSpinBox" name="amplitude_edit">
                 <property name="styleSheet">
                  <string notr="true">QSpinBox {
    /* Background color */
    background-color: #f0f0f0;
    /* Border */
    border: 1px solid #3498db;
    border-radius: 5px;
    /* Text color */
    color: #333;
    /* Padding */
    padding: 1px;
}</string>
                 </property>
                 <property name="maximum">
                  <number>100000</number>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="styleSheet">
                  <string notr="true">QLabel{
font: 75 italic 10pt &quot;Century Gothic&quot;;}</string>
                 </property>
                 <property name="text">
                  <string>Częstotliwość
sygnału[Hz]:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QSpinBox" name="frequency_edit">
                 <property name="styleSheet">
                  <string notr="true">QSpinBox {
    /* Background color */
    background-color: #f0f0f0;
    /* Border */
    border: 1px solid #3498db;
    border-radius: 5px;
    /* Text color */
    color: #333;
    /* Padding */
    padding: 1px;
}</string>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>100000</number>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="styleSheet">
                  <string notr="true">QLabel{
font: 75 italic 10pt &quot;Century Gothic&quot;;}</string>
                 </property>
                 <property name="text">
                  <string>Faza[° ]:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QSpinBox" name="phase_edit">
                 <property name="styleSheet">
                  <string notr="true">QSpinBox {
    /* Background color */
    background-color: #f0f0f0;
    /* Border */
    border: 1px solid #3498db;
    border-radius: 5px;
    /* Text color */
    color: #333;
    /* Padding */
    padding: 1px;
}</string>
                 </property>
                 <property name="minimum">
                  <number>-10000</number>
                 </property>
                 <property name="maximum">
                  <number>100000</number>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="styleSheet">
                  <string notr="true">QLabel{
font: 75 italic 10pt &quot;Century Gothic&quot;;}</string>
                 </property>
                 <property name="text">
                  <string>Częstotliwość
 próbkowania
[Hz]:</string>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::AutoText</enum>
                 </property>
                 <property name="scaledContents">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QSpinBox" name="sampling_frequency_edit">
                 <property name="styleSheet">
                  <string notr="true">QSpinBox {
    /* Background color */
    background-color: #f0f0f0;
    /* Border */
    border: 1px solid #3498db;
    border-radius: 5px;
    /* Text color */
    color: #333;
    /* Padding */
    padding: 1px;
}</string>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>100000</number>
                 </property>
                 <property name="value">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item row="8" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="styleSheet">
                  <string notr="true">QLabel{
font: 75 italic 10pt &quot;Century Gothic&quot;;}</string>
                 </property>
                 <property name="text">
                  <string>Liczba próbek:</string>
                 </property>
                </widget>
               </item>
               <item row="9" column="0">
                <widget class="QSpinBox" name="samples_number_edit">
                 <property name="styleSheet">
                  <string notr="true">QSpinBox {
    /* Background color */
    background-color: #f0f0f0;
    /* Border */
    border: 1px solid #3498db;
    border-radius: 5px;
    /* Text color */
    color: #333;
    /* Padding */
    padding: 1px;
}</string>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>1000</number>
                 </property>
                 <property name="value">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item row="10" column="0">
                <widget class="QPushButton" name="add_button">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dodaj składową sygnału z powyższych danych&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    /* Set the default appearance for the push button */
    background-color: #3498db;
    border: 2px solid #3498db;
    border-radius: 5px;
    color: #fff;
    padding: 8px 12px;
    font-weight: bold;
}

QPushButton:hover {
    /* Change style when the mouse hovers over the button */
    background-color: #207dbb;
    border: 2px solid #207dbb;
}

QPushButton:pressed {
    /* Change style when the button is pressed */
    background-color: #266197;
    border: 2px solid #266197;
}</string>
                 </property>
                 <property name="text">
                  <string>Dodaj</string>
                 </property>
                </widget>
               </item>
               <item row="11" column="0">
                <widget class="QTextBrowser" name="signal_label">
                 <property name="styleSheet">
                  <string notr="true">QTextBrowser {
    /* Background color */
    background-color: #f0f0f0;
    /* Border */
    border: 1px solid #3498db;
    border-radius: 5px;
    /* Text color */
    color: #333;
    /* Padding */
    padding: 1px;
}</string>
                 </property>
                 <property name="readOnly">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="12" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Krok między kolejnymi wartościami w zestawie danych osi OX&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel{
font: 75 italic 10pt &quot;Century Gothic&quot;;}</string>
                 </property>
                 <property name="text">
                  <string>Krok czasowy[s]:</string>
                 </property>
                </widget>
               </item>
               <item row="13" column="0">
                <widget class="QLineEdit" name="time_step_edit">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit {
    /* Background color */
    background-color: #f0f0f0;
    /* Border */
    border: 1px solid #3498db;
    border-radius: 5px;
    /* Text color */
    color: #333;
    /* Padding */
    padding: 1px;
}</string>
                 </property>
                 <property name="text">
                  <string>0.0001</string>
                 </property>
                 <property name="frame">
                  <bool>false</bool>
                 </property>
                 <property name="cursorMoveStyle">
                  <enum>Qt::LogicalMoveStyle</enum>
                 </property>
                </widget>
               </item>
               <item row="14" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="toolTip">
                  <string>Wpisz numer składowej sygnału, na której chcesz wykonywać operacje 
Numer składowej &gt;=1</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel{
font: 75 italic 10pt &quot;Century Gothic&quot;;}</string>
                 </property>
                 <property name="text">
                  <string>Numer składowej:</string>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::AutoText</enum>
                 </property>
                 <property name="scaledContents">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="15" column="0">
                <widget class="QLineEdit" name="signal_number_edit">
                 <property name="styleSheet">
                  <string notr="true">QLineEdit {
    /* Background color */
    background-color: #f0f0f0;
    /* Border */
    border: 1px solid #3498db;
    border-radius: 5px;
    /* Text color */
    color: #333;
    /* Padding */
    padding: 1px;
}</string>
                 </property>
                 <property name="text">
                  <string>1</string>
                 </property>
                 <property name="frame">
                  <bool>false</bool>
                 </property>
                 <property name="cursorMoveStyle">
                  <enum>Qt::LogicalMoveStyle</enum>
                 </property>
                </widget>
               </item>
               <item row="16" column="0">
                <widget class="QPushButton" name="delete_signal_button">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Usuń daną składową sygnału&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    /* Set the default appearance for the push button */
    background-color: #3498db;
    border: 2px solid #3498db;
    border-radius: 5px;
    color: #fff;
    padding: 8px 12px;
    font-weight: bold;
}

QPushButton:hover {
    /* Change style when the mouse hovers over the button */
    background-color: #207dbb;
    border: 2px solid #207dbb;
}

QPushButton:pressed {
    /* Change style when the button is pressed */
    background-color: #266197;
    border: 2px solid #266197;
}</string>
                 </property>
                 <property name="text">
                  <string>Usuń</string>
                 </property>
                </widget>
               </item>
               <item row="17" column="0">
                <widget class="QRadioButton" name="wave_from_file">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Zaznacz, jeżeli chcesz zastąpić powyższy sygnał, danymi z pliku&lt;br/&gt;Akceptowane formaty: .wav&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string>Sygnał z pliku</string>
                 </property>
                </widget>
               </item>
               <item row="18" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="styleSheet">
                  <string notr="true">QLabel{
font: 75 italic 10pt &quot;Century Gothic&quot;;}</string>
                 </property>
                 <property name="text">
                  <string>Plik:</string>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::RichText</enum>
                 </property>
                 <property name="scaledContents">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="22" column="0">
                <widget class="QTextEdit" name="file_name">
                 <property name="styleSheet">
                  <string notr="true">QTextEdit {
    /* Background color */
    background-color: #f0f0f0;
    /* Border */
    border: 1px solid #3498db;
    border-radius: 5px;
    /* Text color */
    color: #333;
    /* Padding */
    padding: 1px;
}</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="acceptRichText">
                  <bool>true</bool>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="31" column="0" colspan="2">
                <widget class="QPushButton" name="print_button">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aktualizuj wykresy wprowadzonymi danymi&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    /* Set the default appearance for the push button */
    background-color: #3498db;
    border: 2px solid #3498db;
    border-radius: 5px;
    color: #fff;
    padding: 8px 16px;
    font-weight: bold;
}

QPushButton:hover {
    /* Change style when the mouse hovers over the button */
    background-color: #207dbb;
    border: 2px solid #207dbb;
}

QPushButton:pressed {
    /* Change style when the button is pressed */
    background-color: #266197;
    border: 2px solid #266197;
}</string>
                 </property>
                 <property name="text">
                  <string>Aktualizuj</string>
                 </property>
                </widget>
               </item>
               <item row="27" column="0" colspan="2">
                <widget class="QPushButton" name="dft_button">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pokaż okno z wykresem DFT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    /* Set the default appearance for the push button */
    background-color: #3498db;
    border: 2px solid #3498db;
    border-radius: 5px;
    color: #fff;
    padding: 8px 16px;
    font-weight: bold;
}

QPushButton:hover {
    /* Change style when the mouse hovers over the button */
    background-color: #207dbb;
    border: 2px solid #207dbb;
}

QPushButton:pressed {
    /* Change style when the button is pressed */
    background-color: #266197;
    border: 2px solid #266197;
}</string>
                 </property>
                 <property name="text">
                  <string>DFT</string>
                 </property>
                </widget>
               </item>
               <item row="29" column="0" colspan="2">
                <widget class="QPushButton" name="dct_button">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pokaż okno z wykresem DCT&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    /* Set the default appearance for the push button */
    background-color: #3498db;
    border: 2px solid #3498db;
    border-radius: 5px;
    color: #fff;
    padding: 8px 16px;
    font-weight: bold;
}

QPushButton:hover {
    /* Change style when the mouse hovers over the button */
    background-color: #207dbb;
    border: 2px solid #207dbb;
}

QPushButton:pressed {
    /* Change style when the button is pressed */
    background-color: #266197;
    border: 2px solid #266197;
}</string>
                 </property>
                 <property name="text">
                  <string>DCT</string>
                 </property>
                </widget>
               </item>
               <item row="26" column="0">
                <widget class="QPushButton" name="choose_file_button">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Otwórz eksplorator plików w celu wyboru pliku&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    /* Set the default appearance for the push button */
    background-color: #3498db;
    border: 2px solid #3498db;
    border-radius: 5px;
    color: #fff;
    padding: 8px 16px;
    font-weight: bold;
}

QPushButton:hover {
    /* Change style when the mouse hovers over the button */
    background-color: #207dbb;
    border: 2px solid #207dbb;
}

QPushButton:pressed {
    /* Change style when the button is pressed */
    background-color: #266197;
    border: 2px solid #266197;
}</string>
                 </property>
                 <property name="text">
                  <string>Wybierz plik</string>
                 </property>
                </widget>
               </item>
               <item row="28" column="0" colspan="2">
                <widget class="QPushButton" name="idft_button">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pokaż okno z wykresem IDFT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    /* Set the default appearance for the push button */
    background-color: #3498db;
    border: 2px solid #3498db;
    border-radius: 5px;
    color: #fff;
    padding: 8px 16px;
    font-weight: bold;
}

QPushButton:hover {
    /* Change style when the mouse hovers over the button */
    background-color: #207dbb;
    border: 2px solid #207dbb;
}

QPushButton:pressed {
    /* Change style when the button is pressed */
    background-color: #266197;
    border: 2px solid #266197;
}</string>
                 </property>
                 <property name="text">
                  <string>IDFT</string>
                 </property>
                </widget>
               </item>
               <item row="30" column="0" colspan="2">
                <widget class="QPushButton" name="idct_button">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pokaż okno z wykresem IDCT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    /* Set the default appearance for the push button */
    background-color: #3498db;
    border: 2px solid #3498db;
    border-radius: 5px;
    color: #fff;
    padding: 8px 16px;
    font-weight: bold;
}

QPushButton:hover {
    /* Change style when the mouse hovers over the button */
    background-color: #207dbb;
    border: 2px solid #207dbb;
}

QPushButton:pressed {
    /* Change style when the button is pressed */
    background-color: #266197;
    border: 2px solid #266197;
}</string>
                 </property>
                 <property name="text">
                  <string>IDCT</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="layout1">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
